d_decimation: 1.0